test/cpp/grpclb/grpclb_test.cc

```diff
@@ -0,0 +1,688 @@
+/*
```

Comment:
It looks like this test mainly uses the C-core API, not the C++ API.  Given that, why is it in test/cpp instead of test/core?  Is it just because of the C++ code used to access the protobufs?